### PR TITLE
More python3 and general fixups

### PR DESCRIPTION
**Pull request by sfayer**
pylint3 did find a number of things that the other compatibility tests missed...

BEGINRELEASENOTES
FIX: More python3 compatibility tweaks (and other tidying up)
ENDRELEASENOTES

### CODE REVIEW

File: VMDIRAC/WorkloadManagementSystem/Service/VirtualMachineManagerHandler.py [hunk 1]
```diff
@@ -484,7 +484,7 @@ def export_getAllInfoForUniqueID(self, uniqueID):
     return res
 
   types_getInstancesContent = [dict, (list, tuple),
-                               (int, long), (int, long)]
+                               int, int]
```
Comment (fstagni) [inline VMDIRAC/WorkloadManagementSystem/Service/VirtualMachineManagerHandler.py]: for `(int, long)` you can use `six.integer_types`

**Review by fstagni — COMMENTED** [review]
Apart from the minor comment on using six.integer_types, you can go one step further and:
- provide a environment-py3.yml file
- use it in an action similar to https://github.com/DIRACGrid/DIRAC/blob/integration/.github/workflows/basic-python3.yml

**Comment by sfayer** [comment]
Thanks, I've put the integer_types in, added a python3 environment action that runs the pytest & pylint stuff and then fixed up a couple more issues that it found. Could you please review again?

File: environment-py3.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  - ipython
+  - mock
+  - parameterized
+  - pylint >=1.6.5
+  - pytest >=3.6
+  - pytest-cov >=2.2.0
+  - pytest-mock
+  # code
+  - boto3
+  - apache-libcloud
```
Comment (fstagni) [inline environment-py3.yml:25]: Looks like `boto3` is part of DIRACOS but not `apache-libcloud`. You can simply add it to https://github.com/DIRACGrid/DIRACOS/blob/master/config/requirements.txt

**Review by fstagni — APPROVED** [review]

**Pull request merged**